File: Dante1/Module1.07/Zadanie7.02/main.c
#include <stdio.h>
#include <math.h>

int sqr(int a);

int main()
{
    int number;

    printf("Enter radix: ");
    int check = scanf("%i",&number);
    if(check != 1)
    {
        printf("Incorrect input\n");
        return 1;
    }

    int result = sqr(number);
    if(result == -1)
    {
        printf("Incorrect input\n");
        return 1;
    }
    printf("Result is: %i",result);
    return 0;
}

int sqr(int a)
{
    return a * a;
}

File: Dante1/Module1.07/Zadanie7.02/main.rs
fn sqrt(number: i32) -> i32
{
    number * number
}

fn main()
{
    let mut input: String = String::new();
    println!("Enter: ");

    std::io::stdin().read_line(&mut input).expect("Incorrect input");
    let number: i32 = input.trim().parse().expect("Incorrect parsing");

    println!("Result is: {}", sqrt(number));
}
